packages/governance/src/paramGovernance/paramManager.js

```diff
@@ -290,13 +288,15 @@ const makeParamManagerBuilder = zoe => {
       getString: name => getTypedParam(ParamType.STRING, name),
       getUnknown: name => getTypedParam(ParamType.UNKNOWN, name),
       getVisibleValue,
-      getParamList,
       getInternalParamValue,
-      ...updateFunctions,
+      // Getters and setters for each param value
+      ...getters,
+      ...setters,
+      // Collection of all getters for passing to read-only contexts
+      asGetters: () => getters,
```

Comment:
to support https://github.com/Agoric/agoric-sdk/issues/4784, in another PR

Reply:
@Chris-Hibbert WDYT of `selectGetters()` ? since it returns a subset of the methods of the param manager

Reply:
I want to keep that it's a subset interface on the full manager. "param" is implied by the paramManager and "all" seems uninformative. So IMO,
`asGetters` > `selectGetters` > `paramGetters` > `allGetters`

Reply:
> it doesn't work as the name the callers would see when they have a paramManager: pm.selectGetters().setFoo(bar)

Right now the getters are on both:
```
type TypedParamManager = ParamManagerBase & Getters<T> & Updaters<T> & {selectGetters: () => Getters<T>}}
```
 I expected `pm.getFoo()` is what someone holding `pm` would use and the `asGetters()` would be only for passing along in another object.

For now I'll make that `selectGetters` but another improvement we could consider is having all the "reads" be on a sub-object that can be passed. Then there's a clear distinction in the powers of the full object and its sub.
```
type TypedParamManager = Updaters<T> & {read: Getters<T>}}
```
To do that we'll need to kill the get[typename] style methods of `ParamManagerBase`. I don't want to saddle this PR with that but if you think it's worth doing I'll open a ticket.

Reply:
Hey, I kind of like `readonly` for what this is doing. Though it has to be a method: 207bc70d3